source/markdown/posts/another.md

```diff
@@ -0,0 +1,5 @@
+#### Another post
```

Comment:
I think these are fine to name as `1.md`, `2.md` and `3.md` so that Amira doesn't have to change it in the main file as the blog posts change